sale_automatic_workflow_advance/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+* Sale Automatic Workflow Advance
```

Comment:
Could you improve a little bit this and add a USAGE. Thanks

Reply:
@rousseldenis code updated base your comments.